toolkit/motors/rev_motors.py

```diff
@@ -69,7 +64,6 @@ def __init__(self, can_id: int, inverted: bool = True, brushless: bool = True, c
             brushless (bool, optional): Whether the motor is brushless. Defaults to True.
             config (SparkMaxConfig, None): The configuration for the motor controller. Defaults to None.
         """
-        super().__init__()
```

Comment:
Can you explain why we got rid of the super call?

Reply:
At the time, the tests were not working, so I removed the parent init to see if that made any changes. There is no class init for the parent, so I didn't put it back as to not confuse anyone from thinking it did anything. 

Now that I am thinking about it however, It might be good for logging, so Ill make some changes and update the branches